widgets/src/components/common/Table/custom/ActionCell.js

```diff
@@ -5,21 +5,28 @@ import AssessmentIcon from '@material-ui/icons/Assessment';
 
 import IconMenuButton from 'components/common/IconMenuButton';
 
-const ActionCell = () => {
-  const menuItems = [
-    {
-      text: 'Claim',
-      icon: <HowToRegIcon fontSize="small" />,
-    },
-    {
+const ActionCell = (options, openDiagram) => data => {
+  const menuItems = [];
+
+  if (options.showComplete) {
+    menuItems.push({
       text: 'Complete',
       icon: <DoneIcon fontSize="small" />,
-    },
-    {
-      text: 'Diagram',
-      icon: <AssessmentIcon fontSize="small" />,
-    },
-  ];
+    });
+  }
+
+  if (options.showClaim) {
+    menuItems.push({
+      text: 'Claim',
```

Comment:
is this possible to use i18n in that 'text' string here? if it is, pls do integrate and also some other parts in your branch. tnx